requirements.txt

```diff
@@ -45,3 +45,4 @@ wandb>=0.16.4
 # it is for CLI usage ONLY
 yamllint>=1.35.1
 referencing>=0.35.0
+prettytable>=3.10.0
```

Comment:
What is the impact of adding this dependency - is this something that we've cleared?